src/sentry/utils/pytest/fixtures.py

```diff
@@ -139,6 +139,18 @@
 }
 
 
+def django_db_all(func=None, *, transaction=None, **kwargs):
+    """Pytest decorator for resetting all databases"""
+
+    if func is not None:
+        return pytest.mark.django_db(func)
```

Comment:
I would have expected this to also get `databases="__all__"`, but I might be missing something.

Reply:
You're totally right.